### ISSUE TITLE
Menu/navigation bar

### ISSUE THREAD

**Issue by LouiseRand** [LouiseRand/studentzone#9]

**Comment by LouiseRand** [comment]
- [x] The user should be able to click on the items/links to be directed to the appropriate pages.
- [x] The items/links in the menu bar should be properly spaced apart to allow for accessibility and improve overall design.
- [x] #8
